qutebrowser/browser/hints.py

```diff
@@ -860,6 +865,8 @@ def _fire(self, keystr):
                 return
             handler = functools.partial(url_handlers[self._context.target],
                                         url, self._context)
+            if self._context.chronicle:
+                objreg.get('web-history').add_from_tab(url, url, "")
```

Comment:
You can simply use the `add_url` method of `WebHistory` instead - `add_from_tab` handles url/requested_url correctly, which we don't need to do here.